server/opts.go

```diff
@@ -1286,6 +1286,22 @@ func (o *Options) processConfigFileLine(k string, v any, errors *[]error, warnin
 			opFiles = append(opFiles, v)
 		case []string:
 			opFiles = append(opFiles, v...)
+		case []any:
```

Comment:
You could have a look at the helper `parseStringArray()`. That could simplify the `case "operator"` a bit, I think.

Reply:
@Jarema Do you want to look at using that helper or are we good as is?